lib/simplifyExpression/fractionsSearch/cancelLikeTerms.js

```diff
@@ -288,10 +290,14 @@ function cancelTerms(numerator, denominator) {
     return new CancelOutStatus(numerator, denominator, true);
   }
 
-  // case 4: the numerator is a constant and denominator is a polynomial term
+  // case 4: the numerator is a constant and denominator is a polynomial term that has a coefficient
+  // or is multiplication node
   // e.g. 2 / 4x -> 1 / 2x
+  // e.g. ignore cases like:  2 / a and 2 / x^2
```

Comment:
I think (@aliang8 correct me if I'm wrong) the problem was, if there is no coefficient in the denominator, then line 308 `const frac = Node.Creator.operator('/', [numerator, coeff]);` was failing with error "Array containing Nodes expected for parameter "args"". Because `coeff` is null or undefined or something.